vllm/utils.py

```diff
@@ -54,6 +54,22 @@ def reset(self) -> None:
         self.counter = 0
 
 
+class Matmul(torch.nn.Module):
```

Comment:
consider moving these definitions to a hpu specific file, vllm/hpu_utils.py , or maybe vllm/hpu/ops.py since these are ops.